nextcord/polls.py

```diff
@@ -0,0 +1,439 @@
+
+        return payload
+
+    @classmethod
+    def from_dict(cls, data: PollAnswerPayload) -> Self:
+        # data['answer_id'] is always sent as part of Discord's HTTP/Gateway
+        return cls(answer_id=data["answer_id"], poll_media=PollMedia.from_dict(data["poll_media"]))  # type: ignore[reportTypedDictNotRequiredAccess]
+
+
+class PollAnswerCount:
+    """The answer count for an answer in a poll.
+
+    .. versionadded:: 3.0
+
+    Attributes
+    ----------
+    id: :class:`int`
+        The ID of the answer in question.
+    me_voted: :class:`bool`
+        Whether the bot has voted for this option.
+    count: :class:`int`
+        The number of users who have voted for this option.
+    """
+
+    __slots__ = ("id", "me_voted", "count", "poll")
+
+    def __init__(self, data: PollAnswerCountPayload, poll: Poll) -> None:
+        self.poll: Poll = poll
+        self.id: int = data["id"]
+        self.me_voted: bool = data["me_voted"]
+        self.count: int = data["count"]
+
+    def __repr__(self) -> str:
+        return (
+            "<PollAnswerCount "
+            f"id={self.id!r} "
+            f"me_voted={self.me_voted!r} "
+            f"count={self.count!r} "
+            ">"
+        )
+
+    def voters(
+        self, *, limit: Optional[int] = None, after: Optional[Snowflake] = None
+    ) -> AnswerVotersIterator:
+        """Returns an :class:`AsyncIterator` representing the users who have voted for this option.
+
+        The ``after`` parameter must represent an user
+        and meet the :class:`abc.Snowflake` abc.
+
+        Examples
+        --------
+        Usage ::
+
+            # I do not actually recommend doing this.
+
+            async for voter in count.voters():
+                await channel.send(f"{voter} has voted with choice number {count.id}!")
+
+        Flattening into a list ::
+
+            voters = await count.voters().flatten()
+            # voters is now a list of User...
+            await channel.send("Those people have voted with choice number {count.id}: {', '.join(voters)}")
+
+
+        Parameters
+        ----------
+
+        limit: Optional[:class:`int`]
+            The maximum number of results to return.
+            If not provided, returns all the users who
+            voted to the choice.
+        after: Optional[:class:`abc.Snowflake`]
+            For pagination, votes are sorted by member.
+        """
+        if limit is None:
+            limit = self.count
+
+        return AnswerVotersIterator(
+            message=self.poll.message, answer_id=self.id, after=after, limit=limit
+        )
+
+    def to_dict(self) -> PollAnswerCountPayload:
+        payload: PollAnswerCountPayload = {
+            "id": self.id,
+            "me_voted": self.me_voted,
+            "count": self.count,
+        }
+        return payload
+
+
+class PollResults:
+    """The result of the poll.
+    This contains the numbers of vote for each answer.
```

Comment:
I know it's quite pedantic, but multiline docstrings have a blank line between the summary and description
```python
"""Summary goes here.

Description here.
"""
```